templates/Apple/ResourceFork.tcl

```diff
@@ -58,6 +72,16 @@ section "File Format" {
                 incr i
             }
         }
+
+        goto [expr {$map_offset + $name_list_offset}]
+        section "Name List" {
+            for {set i 0} {![end]} {incr i} {
+                section "Name $i" {
+                    set num_name_bytes [uint8 "Length"]
```

Comment:
are these utf8? or macroman?

Reply:
Most likely Mac Roman.  Does HexFiend use the TCL encodings and their names (`encoding names` in tclsh)?  If so I can replace the `ascii` below with `str $num_name_bytes "macRoman" "Name"`.

Reply:
I tested it out and switched from `ascii ...` to `str ... "macRoman" ...`.